pages/ppr-marketing.vue

```diff
@@ -1,25 +1,25 @@
 <template>
   <div>
-    <header class="ppr-bg auth d-flex align-center">
+    <header class="hero-banner d-flex align-center">
```

Comment:
Consistent naming for easier development.

Also, this header is indeed a [hero banner](https://www.google.com/search?q=hero+banner), not a "background".